ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/execution/ListQueriesExecutorTest.java

```diff
@@ -39,17 +41,39 @@
 import io.confluent.ksql.serde.KeyFormat;
 import io.confluent.ksql.statement.ConfiguredStatement;
 import io.confluent.ksql.util.PersistentQueryMetadata;
+
+import java.util.Collections;
+import java.util.Map;
 import java.util.Optional;
+
+import org.apache.kafka.streams.KafkaStreams;
+import org.junit.Before;
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
 @RunWith(MockitoJUnitRunner.class)
 public class ListQueriesExecutorTest {
 
+  private static final KsqlHostInfoEntity LOCAL_HOST = new KsqlHostInfoEntity("some host", 555);
+  private static final KafkaStreams.State QUERY_STATE = KafkaStreams.State.RUNNING;
+  private static final Map<KsqlHostInfoEntity, String> KSQL_HOST_INFO_MAP = Collections.singletonMap(LOCAL_HOST, QUERY_STATE.toString());
   @Rule public final TemporaryEngine engine = new TemporaryEngine();
 
+  @Mock
+  SessionProperties sessionProperties;
+
+  KafkaStreamsStateCount kafkaStreamsStateCount;
```

Comment:
made it private, but I don't think this can be mocked. `shouldListQueriesBasic` test compares the state count object created in the output of `ListQueriesExecutor` with this variable and if it's a mock, it wouldn't be a match.